data/enterprise-nav-data.json

```diff
@@ -204,8 +204,8 @@
             "path": "admin/infrastructure/backup-restore"
           },
           {
-            "title": "Active/Active",
```

Comment:
I'm thinking through this, and I'm wondering if just "Admin" will be confusing for users, since this is underneath a broader section called administration. What if we called it Admin CLI Commands or something similar? That's really what this is about, right? I just want to make sure we're giving users as much information as possible from the name so they know what they're going to get when they click.

Reply:
Admin CLI Commands sounds better. Thanks!